lib/core/Doc.js

```diff
@@ -49,11 +50,11 @@ class Doc extends React.Component {
       editLink = (
         <a
           className="edit-page-link button"
-          href={
-            this.props.config.translationRecruitingLink +
-            '/' +
+          href={urljoin(
+            this.props.config.translationRecruitingLink,
+            '/',
```

Comment:
The point of using url-join is so that we don't need to manually concatenate the string with '/'

Example: 

To get `https://crowdin.com/project/docusaurus/es-ES`, we are doing
```js
this.props.config.translationRecruitingLink + '/' + this.props.language;
```
Using url-join, we can do

```js
urljoin(this.props.config.translationRecruitingLink, this.props.language);
```

Please refactor similar codes 😄

Reply:
Sure 👍